Source/WebKit/WebProcess/Plugins/PDF/PDFPluginBase.h

```diff
@@ -296,6 +296,37 @@ class PDFPluginBase : public ThreadSafeRefCounted<PDFPluginBase>, public WebCore
     RefPtr<WebCore::Element> m_annotationContainer;
     bool m_pdfDocumentWasMutated { false };
 
+    // Set overflow: hidden on the annotation container so <input> elements scrolled out of view don't show
+    // scrollbars on the body. We can't add annotations directly to the body, because overflow: hidden on the body
+    // will break rubber-banding.
+    static constexpr auto annotationStyle =
+    "#annotationContainer {"
+    "    overflow: hidden; "
+    "    position: absolute; "
+    "    pointer-events: none; "
+    "    top: 0; "
+    "    left: 0; "
+    "    right: 0; "
+    "    bottom: 0; "
+    "    display: -webkit-box; "
```

Comment:
At some point we should stop using deprecated flexbox.

Reply:
Is `display: flex;` an equivalent replacement?

Reply:
I don't believe it is completely equivalent in all scenarios and there are some differences. (e.g. usage of `-webkit-line-clamp`). But perhaps for our purposes it could be!